.github/workflows/rune_centos_with_docker_and_crictl.yml

```diff
@@ -60,13 +66,12 @@ jobs:
         cd /root/inclavare-containers/rune/libenclave/internal/runtime/pal/skeleton;
         make -j${CPU_NUM} && cp liberpal-skeleton-v*.so /usr/lib;
         cat >Dockerfile <<-EOF
-        FROM centos:8.1.1911
+        FROM scratch
 
-        RUN mkdir -p /run/rune
-        WORKDIR /run/rune
+        COPY encl.bin /
+        COPY encl.ss /
 
-        COPY encl.bin .
-        COPY encl.ss .
+        ENTRYPOINT [\"sh\"]
```

Comment:
Actually skeleton doesn't have a real entry point to run. In order to reveal this fact, it is recommended to intent to explicitly set a "dummy" in ENTRYPOINT.

Reply:
> Actually skeleton doesn't have a real entry point to run. In order to reveal this fact, it is recommended to intent to explicitly set a "dummy" in ENTRYPOINT.

Done.